### ISSUE TITLE
Graphviz Required to run View

### ISSUE THREAD

**Issue by jamesnyika** [ztellman/automat#26]
Thanks for this amazing library Zach. 
You may want to mention in the README.md that automat uses rhizome which in turn leverages Graphviz and that you need to have it installed first. If not for that fact that I suspected you were using other libraries of your own, I was stuck looking at a java error on UnixProcesses and no idea why! 

 Thanks again

**Comment by ztellman** [comment]
Yeah, this is very explicit in the [rhizome readme](https://github.com/ztellman/rhizome), but not here.  I'll leave this open until that's resolved.
